### ISSUE TITLE
Deploy compiled artifacts to Maven Central and JCenter

### ISSUE THREAD

**Issue by jblaszczyk** [alpha-asp/Alpha#148]
Currently the only way to use the solver is to clone it and build it. It would be far easier to use in projects by importing it from central repositories via maven or gradle like any other library. This could help increase usage and feedback from users.

**Comment by lorenzleutgeb** [comment]
Yes, that you for raising the issue. I wanted to do this for a long time, but would like to resolve #108 first.
